index.html

```diff
@@ -43,6 +44,188 @@ <h3 class="motto" >
             </div>
             
     </section>
+  <!-- finish-mobile-version project starts here-->
+  <section class="works">
+    <div class="card-one">
+  <a href=""><img class="img-1" src="images\Snapshoot Portfolio.png" alt=""></a>
+  <h1>
+    Tonic
+  </h1>
```

Comment:
Please use a different header tag`(h2, h3, h4, h5)` here instead of the `h1`. It's recommended to have just one `h1` tag on each page which is the main heading.